tests/shared/src/test/scala/scala/meta/tests/quasiquotes/ErrorSuite.scala

```diff
@@ -813,10 +813,9 @@ class ErrorSuite extends FunSuite {
       q"def foo(...$paramss)(y: Int) = ???"
     """).replace("\r", ""),
       """
-      |<macro>:5: implementation restriction: can't mix ...$ with anything else in parameter lists.
-      |See https://github.com/scalameta/scalameta/issues/406 for details.
+      |<macro>:5: = expected but ( found
```

Comment:
```suggestion
      |<macro>:5: = expected but ( found, can't mix elipsis with additional parameter lists
```
maybe a bit clearer?